src/modules/markets-list/containers/markets-view-container.js

```diff
@@ -29,8 +29,8 @@ const mapStateToProps = (state, { location }) => {
     isMobile: state.appStatus.isMobile,
     markets,
     category,
-    defaultFilter: state.filterOption,
-    defaultSort: state.sortOption
+    defaultFilter: state.filterSortOptions.marketFilter,
```

Comment:
Selector?

Reply:
The complexity of the selector isn't the only consideration. Decoupling the shape of state from the components is also something to consider.